types/prosemirror-view/index.d.ts

```diff
@@ -399,253 +384,256 @@ export class EditorView<S extends Schema = any> {
+     * `decorations` is an array of node or inline decorations that are
+     * active around the node. They are automatically drawn in the
+     * normal way, and you will usually just want to ignore this, but
+     * they can also be used as a way to provide context information to
+     * the node view without adding it to the document itself.
+     */
+    nodeViews?: {
+        [name: string]: (
+            node: ProsemirrorNode<S>,
+            view: EditorView<S>,
+            getPos: (() => number) | boolean,
+            decorations: Decoration[],
+        ) => NodeView<S>;
+    } | null;
+    /**
+     * The DOM serializer to use when putting content onto the
+     * clipboard. If not given, the result of
+     * [`DOMSerializer.fromSchema`](#model.DOMSerializer^fromSchema)
+     * will be used.
+     */
+    clipboardSerializer?: DOMSerializer<S> | null;
+    /**
+     * A function that will be called to get the text for the current
+     * selection when copying text to the clipboard. By default, the
+     * editor will use [`textBetween`](#model.Node.textBetween) on the
+     * selected range.
+     */
+    clipboardTextSerializer?: ((this: ThisT, p: Slice<S>) => string) | null;
+    /**
+     * A set of [document decorations](#view.Decoration) to show in the
+     * view.
+     */
+    decorations?: ((this: ThisT, state: EditorState<S>) => DecorationSet<S> | null | undefined) | null;
+    /**
+     * When this returns false, the content of the view is not directly
+     * editable.
+     */
+    editable?: ((this: ThisT, state: EditorState<S>) => boolean) | null;
+    /**
+     * Control the DOM attributes of the editable element. May be either
+     * an object or a function going from an editor state to an object.
+     * By default, the element will get a class `"ProseMirror"`, and
+     * will have its `contentEditable` attribute determined by the
+     * [`editable` prop](#view.EditorProps.editable). Additional classes
+     * provided here will be added to the class. For other attributes,
+     * the value provided first (as in
+     * [`someProp`](#view.EditorView.someProp)) will be used.
+     */
+    attributes?:
+        | { [name: string]: string }
+        | ((this: ThisT, p: EditorState<S>) => { [name: string]: string } | null | undefined | void)
+        | null;
+    /**
+     * Determines the distance (in pixels) between the cursor and the
+     * end of the visible viewport at which point, when scrolling the
+     * cursor into view, scrolling takes place. Defaults to 0.
+     */
+    scrollThreshold?: number | { top: number; right: number; bottom: number; left: number } | null;
+    /**
+     * Determines the extra space (in pixels) that is left above or
+     * below the cursor when it is scrolled into view. Defaults to 5.
+     */
+    scrollMargin?: number | { top: number; right: number; bottom: number; left: number } | null;
 }
+export type HandleDOMEventsProp<ThisT = unknown, S extends Schema = any> = Partial<
```

Comment:
This is what was added, to automatically include the event types for the `handleDOMEvents` prop.